docs/faq.md

```diff
@@ -295,11 +295,11 @@ Flash Attention is a feature of most modern models that can significantly reduce
 
 The K/V context cache can be quantized to significantly reduce memory usage when Flash Attention is enabled.
 
-To use quantized K/V cache with Ollama you can set the following environment variable:
-
-- `OLLAMA_KV_CACHE_TYPE` - The quantization type for the K/V cache.  Default is `f16`.
```

Comment:
The default value is missing from the new documentation.

Reply:
is that relevant in context of this is not going to be merged anyway?